docs/team/alanlowzies.md

```diff
@@ -1,6 +1,103 @@
+  `UnknownFileException`.
+- Contributed to the `ExerciseList` API, which is mainly done by my team member Haofeng, by adding
+  additional functionality to the API such as populating the `exercise.txt` file with default exercises. The exercises
+  were thought of by Haofeng :)
+- Wrote JUnit test cases for WerkIt! APIs including (but not limited to) `WorkoutCommand`, `FileManager`, `Parser`, etc.
+- Created skeleton codes and packages for the project.
+
+
+### Enhancements Implemented
+- (Developer and User Guides) Replaced Markdown quote blocks with custom CSS boxes for informational and warning boxes.
+  <br/><br/>**Sample:**<br/>
+<span class="box warning">⚠️This is an example of a warning box. Contains advice or instructions that users
+should take note of to avoid issues in the application.</span>
+<span class="box info">🧾  This is an information box. Contains information that may be useful for
+the users.</span>
+
+### Contributions to the User Guide (UG)
+- Wrote the guide on [creating a new workout](../UserGuide.md#create-a-workout-workout-new).
+- Wrote the informational section on [WerkIt!'s local storage structure](../UserGuide.md#werkits-local-storage-information).
+- Wrote the [Quick Start Guide](../UserGuide.md#quick-start-guide) section, which guides the user on the necessary
+prerequisites to download and the steps to get it running on the user's computer.
+  - Includes a list of recommended OSes, their versions, and the terminals to use for each OS.
+- Wrote the [frequently asked questions](../UserGuide.md#frequently-asked-questions-faq) section.
+
+### Contributions to the Developer Guide (DG)
+- Wrote the following contributions to the ['Implementation'](../DeveloperGuide.md#implementation) section:
+    - How WerkIt! repeatedly waits and receives user inputs ([link to part](../DeveloperGuide.md#getting-user-input-continuously)).
+    - How WerkIt! parses user inputs and determines the appropriate procedures to take 
+      ([link to part](../DeveloperGuide.md#parsing-user-input-and-getting-the-right-command)).
+    - How WerkIt! processes and creates a new workout when requested by the user
+      ([link to part](../DeveloperGuide.md#create-new-workout)).
+      - Includes sequence diagrams to visualise the process.
+- Wrote the instructions on 
+[how to set up the development environment](../DeveloperGuide.md#setting-up-your-development-environment).
+- Wrote part of the [glossary](../DeveloperGuide.md#glossary)
+- Wrote the [table of contents](../DeveloperGuide.md#table-of-contents).
+
+### Contributions to Team-Based Tasks
+- Set up the team's organisation and repository on GitHub
+- Set up and maintained the team's Google Drive folder, which includes the team's project notebook
+and other documents for the co-requsuite module CS2101 Effective Communication for Computing Professionals.
+- Helped to maintain the team's GitHub repository issue tracker.
```

Comment:
does setting up meeting and attending meeting counts? x)

Reply:
Hmm, I'm not too sure about that 😅